### ISSUE TITLE
[Beta 2022.6.0b0] Errors from Homeassistant.components.hassio and homeassistant.components.hassio.handler

### ISSUE THREAD

**Issue by jweston2112** [home-assistant/supervisor#3656]
### Describe the issue you are experiencing

After updating to Beta 2022.6.0b0 I reecieved the following errors in my  HA Core Log

![image](https://user-images.githubusercontent.com/70152549/170366281-999f8e50-8947-40d4-97a2-df3e3068f866.png)

Home Assistant Core 2022.6.0b0
Home Assistant Supervisor 2022.05.3
Home Assistant OS 8.1
Path to configuration.yaml: /config

### What is the used version of the Supervisor?

 2022.05.3

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### What is the version of your installed operating system?

8.1

### What version of Home Assistant Core is installed?

2022.6.0b0

### Steps to reproduce the issue

1. Restart HA Core 
2.
3.
...

### Anything in the Supervisor logs that might be useful for us?

```txt
-05-25 16:32:05 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 121, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/_compat.py", line 54, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 111, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 139, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 205, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 65, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/addons.py", line 162, in list
    data_addons = [
  File "/usr/src/supervisor/supervisor/api/addons.py", line 164, in <listcomp>
    ATTR_NAME: addon.name,
  File "/usr/src/supervisor/supervisor/addons/model.py", line 135, in name
    return self.data[ATTR_NAME]
  File "/usr/src/supervisor/supervisor/store/addon.py", line 19, in data
    return self.sys_store.data.addons[self.slug]
KeyError: 'a0d7b954_grafana'
22-05-25 16:32:05 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 121, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/_compat.py", line 54, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 111, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 139, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 205, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 65, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/store.py", line 120, in store_info
    ATTR_ADDONS: [
  File "/usr/src/supervisor/supervisor/api/store.py", line 121, in <listcomp>
    self._generate_addon_information(self.sys_addons.store[addon])
  File "/usr/src/supervisor/supervisor/api/store.py", line 81, in _generate_addon_information
    ATTR_ADVANCED: addon.advanced,
  File "/usr/src/supervisor/supervisor/addons/model.py", line 222, in advanced
    return self.data[ATTR_ADVANCED]
  File "/usr/src/supervisor/supervisor/store/addon.py", line 19, in data
    return self.sys_store.data.addons[self.slug]
KeyError: 'a0d7b954_grafana'
22-05-25 16:32:05 INFO (MainThread) [supervisor.store] Loading add-ons from store: 67 all - 0 new - 0 remove
22-05-25 16:32:05 INFO (MainThread) [supervisor.store] Loading add-ons from store: 67 all - 0 new - 0 remove1
```

### Additional information

from the core log
```
2022-05-25 16:32:05 ERROR (MainThread) [homeassistant.components.hassio.handler] /addons return code 500
2022-05-25 16:32:05 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /addons -
2022-05-25 16:32:05 ERROR (MainThread) [homeassistant.components.hassio.handler] /store return code 500
2022-05-25 16:32:05 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /store -
2022-05-25 16:32:07 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.320 seconds (success: True)
2022-05-25 16:37:11 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.311 seconds (success: True)
2022-05-25 16:42:13 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.317 seconds (success: True)
2022-05-25 16:47:15 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.304 seconds (success: True)
2022-05-25 16:52:17 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.302 seconds (success: True)
2022-05-25 16:57:19 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.305 seconds (success: True)
2022-05-25 17:02:21 DEBUG (MainThread) [homeassistant.components.hassio] Finished fetching hassio data in 2.317 seconds (success: True)
```

**Comment by jweston2112** [comment]
Please let me know if you require any additional information.

**Comment by mdegat01** [comment]
If you go to the [addon store](https://my.home-assistant.io/redirect/supervisor_store/) do you see the community addons repository listed? Or is it missing?

**Comment by andrefriasribeiro** [comment]
I'm having the same error message.

**Comment by jweston2112** [comment]
When I go to the Addons Store I can See both the Official and Community Addons I also have a Few added repositories (ESPHome, etc.)

**Comment by jweston2112** [comment]
Home Assistant Core 2022.6.7
Host Operating System | Home Assistant OS 8.2
Update Channel | beta
Supervisor Version | supervisor-2022.06.1
Agent Version | 1.2.1
Docker Version | 20.10.14
Disk Total | 228.5 GB
Disk Used | 11.8 GB
Healthy | true
Supported | true
Board | rpi4-64
Supervisor API | ok
Version API | ok
Installed Add-ons | Samba share (10.0.0), Node-RED (12.0.2), SSH & Web Terminal (11.0.1), MariaDB (2.4.0), ArgonOne Active Cooling (27), ESPHome (2022.6.2), Studio Code Server (5.1.2), Glances (0.16.0), Samba Backup (5.0.0)

Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:120
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22))
First occurred: 12:06:30 PM (1 occurrences)
Last logged: 12:06:30 PM

Failed to to call /store -

Logger: homeassistant.components.hassio.handler
Source: components/hassio/handler.py:254
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22))
First occurred: 12:06:30 PM (1 occurrences)
Last logged: 12:06:30 PM

/store return code 500


Just to update the current Software versions and let you know its still happening...

I have also seen a message like these about /discovery  as well

**Comment by tggman** [comment]
I JUST started having this issue as well. The error popped up after I went through the following restore from backup/update process:
1. Running HAC 2022.6.7 without any issues related to this bug.
2. Restored HAC from a 2022.6.6 backup (for issues related to the HERE Travel integration). I'm pretty sure the error wasn't present at this point but can't say with absolute certainty.
3. Updated back to 2022.6.7. This is when I first saw the error.

Copy of error in log:
```Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:120
Integration: Home Assistant Supervisor (documentation, issues)
First occurred: 9:54:11 AM (2 occurrences)
Last logged: 9:54:11 AM

Failed to to call /addons -
Failed to to call /store -
```
